test/clerk/integration-test.js

```
@@ -0,0 +1,712 @@
+        }
+        // Find the last valid JSON object in the output (the result)
+        let result = null;
+        const lines = output.split(/\r?\n/).reverse();
+        for (const line of lines) {
+            // Try to extract JSON substring from the line
+            const jsonStart = line.indexOf('{');
+            if (jsonStart !== -1) {
+                const jsonStr = line.slice(jsonStart);
+                try {
+                    const obj = JSON.parse(jsonStr);
+                    if (obj && obj.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)